docs/public/debugging-using-devfile.adoc

```diff
@@ -0,0 +1,56 @@
+# Debugging applications using devfiles
+
+ODO uses devfiles to build and deploy applications. More information on devifles : https://redhat-developer.github.io/devfile/[Introduction to devfile]
+
+To enable debugging mode in the application, we need a devfile with debugrun step. Example of a nodejs devfile with a debugrun step
+
+    apiVersion: 1.0.0
```

Comment:
Putting this block in yaml would yield colourful formatting. :wink: 
\`\`\`yaml
code here
\`\`\`